### DISCUSSION TITLE
Add extra product options ( product add on sale) besides product variation

### DISCUSSION THREAD

**Discussion by ttsg99** [saleor/saleor#7787] [Q&A]
Hi there,

Could you please share if we could define extra product options besides the product Variation in current relase of Saleor? 

For example

Product variation: coffee size L: 10$, size: M: 6$

Product extra options ( product add on sale)
+ Chose one options: sugar 100% or sugar 50% or No sugar
+ Multiple choice: add extra topping xyz: 5$, add special milk foam: 4$, add special topping abc: 2$

When customer choose coffee size L, add special milk foam, topping abc . Total price: 10$ + 4$ +2$

Right now look like we could define product Variation( size L , M ) only ?

Many thanks

Reference:
https://vi.wordpress.org/plugins/woo-extra-product-options/

**Comment by karolkielecki** [comment]
@ttsg99 Hi there, 

You can only define your products based on variants for now. You can create all possible variants. We don't have a dedicated feature to cover this use case for now.

**Comment by susanthenerd** [comment]
Is this still the case?
